### DISCUSSION TITLE
CNAME not configured correctly (Tracking Domains)

### DISCUSSION THREAD

**Discussion by charlesmudy** [postalserver/postal#1645] [Help with using Postal]
Hello,

I have been trying to get this CNAME working for Tracking Domains but I keep getting error even though I have the follow setup and working correctly:

- I have created click.domain.com based on this https://docs.postalserver.io/features/click-and-open-tracking and its working correctly as I am able to get the response Hello.
- Added CNAME click.domain.com with VALUE track.mail.domain.com. to my domain.com
- Added domain.com to list of DOMAINS in Postal dashboard and all valid

Trying to add click/tracking to domain.com is the issue I have, wondering what I am missing?

**Comment by willpower232** [comment]
What error are you getting? It sounds like your DNS is correct.

**Reply by charlesmudy** [reply]
I had to delete the DNS then the adding CNAME was accepted, it’s like there is a conflict. Then I added the DNS domain again and it seems to work now.

almost like I must not touch it again. Isn’t  working smoothly at all.

**Reply by willpower232** [reply]
do you definitely have no other records on "click.domain.com"?

**Reply by charlesmudy** [reply]
Just standard NS record from digital ocean. For the domain to work, I obviously added A record ... that was how I could get Helo. response

I was adding CNAME to domain.com which wasn’t working from postal dashboard. In fact, currently I have not added any track.mail.domain.com which seems to be working as it’s seeing the record from somewhere I don’t know

**Reply by charlesmudy** [reply]
@willpower232 the issue is back now again. I am totally lost.

I have a DNS server for click.domain.com so I can do the `RequestHeader set X-Postal-Track-Host: "1"` and `ProxyPreserveHost on` which is working as I mentioned. And now need to setup CNAME in dashboard which I did under domain.com by adding a new CNAME but it's just not picking it and I don't have any way around this.

If I delete click.domain.com ... the click CNAME check works but then there is no server to get the HELO. response for the above.

Any ideas?

**Reply by willpower232** [reply]
Do you mean you've added a new caddy or nginx (or whatever) configuration for your domain separately from Postal that includes the extra header?

Have you added the domain under Tracking Domains in the web UI?
